### PR TITLE
testing/chrome-gnome-shell: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: testing/chrome-gnome-shell/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		-DBUILD_EXTENSION=FALSE \
+		${CMAKE_CROSSOPTS}
+	make
+}
+
+check() {
```
Comment (TBK) [inline testing/chrome-gnome-shell/APKBUILD]:
```suggestion

```

If there is no test suite then please remove the check().

Comment (Cogitri) [reply testing/chrome-gnome-shell/APKBUILD]: Hm, but why? Once they've added tests we can just flick the switch (`options="!check"`) and it'll work.

Comment (TBK) [reply testing/chrome-gnome-shell/APKBUILD]: Will have to ask higher authority on why...

I have had PRs before with the same reasoning to keep it, but I was asked to remove it.

One reason could be that keeping could cause confusion.

Comment (Cogitri) [reply testing/chrome-gnome-shell/APKBUILD]: Ah, alright. I mean I don't mind removing it, but I like to know the reason for my doing :)

**Pull request closed without merging**